How_to_add_a_method.md

```diff
@@ -30,35 +34,35 @@ A Method in KBase is complete only if it is exposed and executable from within t
+    1.  Additional Java runtime dependencies should be added to the [*jars*](https://github.com/kbase/jars) repository following instructions in the readme of that repository. Additional Perl or Python dependencies or other executable dependencies are added by adding the appropriate module to the [*bootstrap*](https://github.com/kbase/bootstrap) repository. After adding to the bootstrap repository, the perl/python or other runtime must be updated locally. A new dependency, however, will not be available on test or production machines until the KBase runtime is rebuilt with your new changes, the last full build being Nov 2014.
+    2.  Persistent data created by the service should be saved to the Workspace or Shock (if saving to Shock, a handle should be created and saved in a Workspace object, which is required in order to view and access the data from the Narrative and landing pages). Provenance data when saving to the Workspace should be set appropriately and provide minimally the service name, service version, method name, input workspace data object references, and input parameters if relatively small.
 8.  Debug your service locally
-    1.  Run it locally and connect to the service using a generated client or even just curl. Services are generally run in Perl with [*Starman*](http://search.cpan.org/~miyagawa/Starman-0.1000/lib/Starman.pm), in Java with [*Glassfish*](https://glassfish.java.net/), and in python with [uWSGI](https://uwsgi-docs.readthedocs.org)/[*WSGI*](http://wsgi.readthedocs.org).
+    1.  Run it locally and connect to the service using a generated client or even just curl. Services are generally run in Perl with [*Starman*](http://search.cpan.org/~miyagawa/Starman-0.1000/lib/Starman.pm), in Java with [*Glassfish*](https://glassfish.java.net/), and in Python with [uWSGI](https://uwsgi-docs.readthedocs.org)/[*WSGI*](http://wsgi.readthedocs.org).
 9.  Write tests for your service
-    1.  Running make test in your module should run unit tests and ideally put up a server, make several calls against the server, and take it down. If the Workspace or other data store is a dependency, the Workspace / data store endpoint should be a configurable parameter, and you should test against next.kbase.us or ci.kbase.us or against a local install of the workspace.
+    1.  Running "make test" in your module should run unit tests and ideally put up a server, make several calls against the server, and take it down. If the Workspace or other data store is a dependency, the Workspace / data store endpoint should be a configurable parameter, and you should test against next.kbase.us or ci.kbase.us or against a local install of the workspace.
```

Comment:
This should be `make test` instead of "make test" so that it renders as code in the HTML version.